docs/tutorials/query-language.md

```diff
@@ -0,0 +1,58 @@
+The Realm JavaScript SDK supports querying based on a language inspired by [NSPredicate](https://realm.io/news/nspredicate-cheatsheet/). 
+
+You query your Realm using the {@link Realm.Collection#filtered Collection.filtered()} method:
+
+```JS
+let dogs = realm.objects('Dog');
```

Comment:
You are absolutely right, I hate that as well and I want to change it everywhere. I'll start here 😄